test/routes/issueRoutesTest.js

```diff
@@ -92,7 +94,7 @@ describe('Issue requests', function(){
                 .get('/issues/1')
                 .then(done, (err) => {
                     expect(err).to.not.be.null;
-                    expect(err).to.have.status(500);                    
+                    expect(err).to.have.status(404);                    
```

Comment:
You've described this test case as `Invalid params`. Params are not invalid here. It's just that issue with id 1 does not exist. This describe have a sentence that describes that issue does not exist.